public/viewjs/consume.js

```diff
@@ -400,9 +415,11 @@ $('#consume-form input').keyup(function(event)
 	Grocy.FrontendHelpers.ValidateForm('consume-form');
 });
 
+//das Timeout wurde eingepflegt damit bei valider vorausgewaehlter Menge die Buttons von selbst aktiviert werden
```

Comment:
For the same reason we keep the public conversations in English, I think we don't need German code or comments...